charts/devlake/values.yaml

```diff
@@ -223,6 +229,8 @@ ui:
 
   extraLabels: {}
 
+  annotations: {}
+
```

Comment:
kindly use podAnnotations

Reply:
updated to `podAnnotations` as requested